### ISSUE TITLE
Plan for publication with fuse

### ISSUE THREAD

**Issue by samreid** [phetsims/circuit-construction-kit-common#500]
From https://github.com/phetsims/circuit-construction-kit-common/issues/463

- [x] 1st round dev testing https://github.com/phetsims/QA/issues/311
- [x] 2nd round dev testing https://github.com/phetsims/QA/issues/314
- [x] code review
- [x] address `// REVIEW` comments, see https://github.com/phetsims/circuit-construction-kit-common/issues/495
- [x] A memory leak was reported in https://github.com/phetsims/circuit-construction-kit-common/issues/499
- [x] design team: signing off on design for fuse reset button https://github.com/phetsims/circuit-construction-kit-common/issues/479 
- [x] design team: discussing play/pause button https://github.com/phetsims/circuit-construction-kit-common/issues/483
- [x] 3rd round dev testing https://github.com/phetsims/QA/issues/361
- [x] rc test

Assigning @ariel-phet and @kathy-phet so they can help assign priority relative to Graphing Quadratics IO 1.1 and Wave Interference with Diffraction screen.

**Comment by ariel-phet** [comment]
@samreid priority is below both of Graphing Quadratics iO and Diffraction screen

**Comment by kathy-phet** [comment]
We need this version - at least a good usable dev version - for use for a class on July 15. @samreid - Can you prioritize the issues that need to be address to make it useable in class, and create a new dev version.

**Comment by samreid** [comment]
I addressed #495 and #499 and they are reassigned to @Denz1994 for review.  After that we can move to dev then RC testing.  I'll co-assign @Denz1994 to this issue so he is aware of priority.

**Comment by Denz1994** [comment]
Issues https://github.com/phetsims/circuit-construction-kit-common/issues/495 and https://github.com/phetsims/circuit-construction-kit-common/issues/499 have been reviewed. Thanks for the changes @samreid. Removing assignment.

**Comment by samreid** [comment]
> We need this version - at least a good usable dev version - for use for a class on July 15.

@kathy-phet and @ariel-phet should I focus on circuit-construction-kit-dc for now or proceed with circuit-construction-kit-dc-virtual-lab at the same time?

**Comment by samreid** [comment]
I published https://phet-dev.colorado.edu/html/circuit-construction-kit-dc/1.1.0-dev.8/phet/circuit-construction-kit-dc_en_phet.html which can be used for dev testing, but will not have time to create QA dev test issue until Wednesday or Thursday.  Assigning to @ariel-phet or @kathy-phet to see if they want to have someone else work on that part.

**Comment by samreid** [comment]
> @kathy-phet and @ariel-phet should I focus on circuit-construction-kit-dc for now or proceed with circuit-construction-kit-dc-virtual-lab at the same time?

Based on the remark in https://github.com/phetsims/QA/issues/314#issuecomment-489875820 I'll go with both.

**Comment by samreid** [comment]
I created the dev test issue, on hold until that's complete.

**Comment by samreid** [comment]
Dev testing complete in https://github.com/phetsims/QA/issues/361, no new issues reported.  @kathy-phet you can use DC or Virtual Lab for the July 15 class:

https://phet-dev.colorado.edu/html/circuit-construction-kit-dc/1.1.0-dev.8/phet/circuit-construction-kit-dc_en_phet.html
https://phet-dev.colorado.edu/html/circuit-construction-kit-dc-virtual-lab/1.1.0-dev.2/phet/circuit-construction-kit-dc-virtual-lab_en_phet.html

**Comment by kathy-phet** [comment]
Thanks, QA and Sam! I'll pass along the links. Does that mean this can also go to RC?

**Comment by samreid** [comment]
Yes, it's ready for RC.  I'll try to create the RC and set up the QA issue by end of business Friday.

**Comment by samreid** [comment]
This is blocked by https://github.com/phetsims/chipper/issues/764#issuecomment-510929071, would be good to hear from @ariel-phet or @jonathanolson before proceeding.

**Comment by samreid** [comment]
https://github.com/phetsims/chipper/issues/764#issuecomment-510929071 is no longer blocking, so I'll proceed with creating the RC.

**Comment by samreid** [comment]
RC test revealed no new problems, I'll deploy.

**Comment by samreid** [comment]
Deployed: https://phet.colorado.edu/sims/html/circuit-construction-kit-dc/latest/circuit-construction-kit-dc_en.html

**Comment by samreid** [comment]
I published both sims, but they haven't appeared on the website yet.  Leaving self-assigned to double check later on.  Also adding @arouinfar as an assignee in case any of the text descriptions need to change.

**Comment by arouinfar** [comment]
@samreid I think it just takes some time for the version to update, I'm seeing 1.1.0 under the About section.

I've updated the Teacher Tips and sim description. Closing.
